client/client.go

```diff
@@ -65,16 +65,6 @@ func New(cfg *config.Config) (*DatabricksClient, error) {
 			},
 			TransientErrors: []string{
 				"REQUEST_LIMIT_EXCEEDED", // This is temporary workaround for SCIM API returning 500.  Remove when it's fixed
```

Comment:
Looking at this sole example remaining, the field could be removed and folded into `ErrorRetriable` as well. In this case in the set of transient errors in `apierr` because it is SCIM related (can be another PR).